File: misc/cgo/test/backdoor/runtime.c
// Copyright 2011 The Go Authors.  All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// Expose some runtime functions for testing.
// Must be in a non-cgo-using package so that
// the go command compiles this file with 6c, not gcc.

// +build gc

typedef char bool;

// This is what a cgo-compiled stub declaration looks like.
void
·Issue7695(struct{void *y[8*sizeof(void*)];}p)
{
	USED(p);
}

File: src/internal/runtime/cgroup/runtime.go
// Copyright 2025 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

package cgroup

import (
	_ "unsafe" // for linkname
)

// Functions below pushed from runtime.

//go:linkname throw
func throw(s string)
